interview-questions.md

```diff
@@ -8,50 +8,111 @@ Then, research the question to expand on your answer. Even if you feel you have
+I have used Git and Github in the beginning of my coding journey, colloborating with others to establish a strong connection in understanding what code I was writing but, also allowing myself to see how other people with different perspectives solving coding challenges in different ways.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Javascript has 6 primitive data types and Boolean is one of them. To recall a Boolean value, a devloper will need to use the relational or comparison operators: >, <, >=, =<, ==, and ===. These operators may be allowed to be paired with other operators like the, Bang Operator (!). Using the above relational or comparison operators will output a true or false statement also known as a Boolean value for the devloper. Usually Boolean values are useful as they are the major foundations that can be bulit up for more complex and logical sets of code blocks.
 
 Researched answer:
+<!-- Referenced https://www.tutorialsteacher.com/javascript/javascript-boolean -->
+Boolean values are either true or false, to illicit these outputs an operator is needed. The Javascript relational and comparison operators are capable of causing the console to release a boolean value; comparison operators: === , and ==; relational operators: >, <, >=, and =<. Depending on if a conditional statement is applicable to a piece of code--the output: "true" will be printed, however if the condtional statement is not applicable to a piece of code--the output: "false" will prtinted. Boolean values permits code to have logic in their structure. More complex patches of code will always use Boolean values. 
+Boolean values are something that I have integrated in my code before to devlop if, else if, and else statements that helps solve nuranced problems. In one of the pratices I have done I have created a simple, but logical piece of code that indeitfies a percentage of an assignement and the the code block was capable of identifying the grade letter. If the score percentage was a 60% the condtional statement i would use to catch this data value would be: 
+
+else if (scorePercentage >= 59 && scorePercentage =< 69){
+    return "Assignements letter grade: D"
+}
 
```

Comment:
This was looking at the categories of operators that produce a boolean value. You have examples of two: equality and relational. Please check out the logical operators also.